src/Marketplace/MarketplaceScreen.js

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
+// SETUP
+import React, { useEffect, useState } from 'react';
 import { View, StyleSheet } from 'react-native';
 import {
-  Title, Text, Button,
+  Title, Text, Subheading, Button, Card,
 } from 'react-native-paper';
 
+// AIRTABLE SETUP
+const Airtable = require('airtable');
+
+const base = new Airtable({ apiKey: 'keyGj20G6MYOtAfqI' })
```

Comment:
It's super important that these hardcoded values don't get pushed to the public repo! Please remove these, put the api key/base key/other values in the .env file and get them from there. For future prs, make sure you use the variable from the .env file and not the actual value for security reasons.